package.json

```diff
@@ -271,57 +84,33 @@
   "xo": {
     "prettier": true,
     "rules": {
-      "@typescript-eslint/naming-convention": "off",
-      "@typescript-eslint/no-base-to-string": "off",
-      "@typescript-eslint/no-floating-promises": "off",
-      "@typescript-eslint/no-implicit-any-catch": "off",
-      "@typescript-eslint/no-require-imports": "off",
-      "@typescript-eslint/no-unsafe-assignment": "off",
-      "@typescript-eslint/no-unsafe-call": "off",
-      "@typescript-eslint/no-unsafe-return": "off",
-      "@typescript-eslint/no-unused-expressions": "off",
-      "@typescript-eslint/no-var-requires": "off",
-      "@typescript-eslint/restrict-plus-operands": "off",
-      "import/extensions": "off",
-      "unicorn/filename-case": "off",
-      "unicorn/prefer-module": "off",
-      "unicorn/prefer-node-protocol": "off"
+      "unicorn/prefer-module": "off"
     }
   },
   "devDependencies": {
-    "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@types/chai": "^4.2.11",
-    "@types/check-types": "^7.3.1",
-    "@types/glob": "^7.1.1",
-    "@types/mocha": "^9.0.0",
-    "@types/node": "^16.0.0",
-    "@types/vscode": "^1.46.0",
-    "chai": "^4.2.0",
-    "check-types": "^11.1.2",
-    "glob": "^7.1.6",
-    "mocha": "^9.0.0",
-    "nyc": "^15.1.0",
+    "@types/tape": "^4.0.0",
+    "@types/vscode": "^1.0.0",
+    "@vscode/test-electron": "^2.0.0",
+    "esbuild": "^0.14.0",
     "prettier": "^2.0.0",
     "remark-cli": "^10.0.0",
+    "remark-language-server": "^1.0.0",
     "remark-preset-wooorm": "^9.0.0",
-    "rimraf": "^3.0.2",
+    "rimraf": "^3.0.0",
+    "tape": "^5.0.0",
     "typescript": "^4.0.0",
     "vsce": "^2.0.0",
-    "vscode-test": "^1.3.0",
+    "vscode-languageclient": "^7.0.0",
+    "vscode-test": "^1.0.0",
     "xo": "^0.47.0"
   },
-  "dependencies": {
-    "npm-module-path": "^2.0.2",
-    "remark": "^13.0.0",
-    "yaml": "^1.10.0"
-  },
   "scripts": {
-    "clean": "rimraf out",
+    "build:language-server": "esbuild node_modules/remark-language-server/index.js --bundle --platform=node --minify --format=cjs --outfile=out/remark-language-server.js",
```

Comment:
ESBuild can’t process `require` statements which load NodeJS builtins, so the language server is built as a CJS module. It is able to use dynamic `import()` this way, so everything works fine this way (this is also tested).